packages/dev/core/src/Shaders/ShadersInclude/screenSpaceRayTrace.fx

```diff
@@ -207,17 +214,25 @@ bool traceScreenSpaceRay1(
 
         // Camera-space z of the scene
         sceneZMax = texelFetch(csZBuffer, ivec2(hitPixel), 0).r;
-
+    #ifdef SSRAYTRACE_SCREENSPACE_DEPTH
+        sceneZMax = -linearizeDepth(sceneZMax, nearPlaneZ, farPlaneZ);

```

Comment:
I don't understand why we have to negate the view depth? Same question for the three other occurrences below.

Reply:
This was because I was testing in RH mode :)
I've moved this inside the linearizeDepth functions now.